src/modules/navigator/mission_base.cpp

```diff
@@ -535,10 +535,8 @@ void MissionBase::setEndOfMissionItems()
 {
 	position_setpoint_triplet_s *pos_sp_triplet = _navigator->get_position_setpoint_triplet();
 
-	if (_land_detected_sub.get().landed) {
-		_mission_item.nav_cmd = NAV_CMD_IDLE;
-
-	} else {
+	// set a loiter item if not landed
```

Comment:
Not sure if needed - maybe we can only do setEndOfMissionItems() if !landed?

Reply:
As is now preserves the previous behavior no?